TripalCultivate/TripalCultivate-Phenotypes
# g0.135 Resolve problems and bring code to standard

**Issue #135 ** Resolve Problems and bring up code to standard.

## Motivation



## What does this PR do?
*Please describe each things this PR does. For example, a PR may 1) solve a specific bug, 2) create an automated test to ensure it doesn't return.*

1. Resolve problems detected by PHP Sniffer & Beautifier, problems like
- Use of \Drupal::service construct (Use DI to resolve this type).
- Comments exceeding 80 characters.
- Unused use statements and variables.
- Required space between foreach and ().

3. Upgrade automated test to use Phenotypes Traits and data provider.

## Testing

### Automated Testing
*Please describe each automated test this PR creates and provide a list of the assertions it makes using casual language.*
*Do not just say things like "asserts the array is not empty" but rather say "Ensures that the return value of method X with these parameters is not an empty array".*

The following tests have been revised
- [ ] test/src/Kernel/**ServiceGenusOntologyTest.php**
- [ ] test/src/Kernel/**ServiceGenusProjectTest.php**
- [ ] test/src/Kernel/**ServiceTermTest**


### Manual Testing
*Describe in detail how someone should manually test this functionality.*
*Make sure to include whether they need to build a docker from scratch, create any records, etc.*

- Load the following files and confirm that for each file the problems tab should show 0 problem.

![image](https://github.com/user-attachments/assets/a28d1b23-18b7-483f-8824-77382b4ea102)

- [ ] src/Controller/**TripalCultivatePhenotypesSettings.php**
- [ ] src/Form/**TripalCultivatePhenotypesOntologySettingsForm.php**
- [ ] src/Form/**TripalCultivatePhenotypesRSettingsForm.php**
- [ ] src/Form/**TripalCultivatePhenotypesWatermarkSettingsForm.php**
- [ ] tests/src/Functional/**ConfigOntologyTermsTest.php**
- [ ] test/src/Functional/**ConfigRRulesTest.php**
- [ ] test/src/Functional/**ConfigWatermarkTest.php**
- [ ] test/src/Functional/**InstallTest.php**
- [ ] test/src/Unit/**ConfigRRulesFormTest.php**
- [ ] test/src/Unit/**ConfigWatermarkFormTest.php**
- [ ] test/src/Kernel/**ServiceGenusOntologyTest.php**
- [ ] test/src/Kernel/**ServiceGenusProjectTest.php**
- [ ] test/src/Kernel/**ServiceTermTest**
